packages/api/src/rpc/payment.ts

```diff
@@ -0,0 +1,18 @@
+const paymentRpc = {
```

Comment:
Just one doubt.. Why do we need to have paymentRpc call, isn't it already available with polkadot/api by default?

Reply:
It should be. I say we recently enabled the payment rpc on cennznet: update/2.0.0 and thought we had to add it like a custom rpc to our api. I will remove it.